radsat-sk/operation/subsystems/RAntenna.c

```diff
@@ -91,7 +92,7 @@ int antennaDeploymentAttempt(void) {
 		if (!RAntennaStatus.DeployedAntennaOne || !RAntennaStatus.DeployedAntennaTwo || !RAntennaStatus.DeployedAntennaThree || !RAntennaStatus.DeployedAntennaFour) {
 
 			// Arm A Side Antenna system
-			error = IsisAntS_setArmStatus(ANTENNA_I2C_SLAVE_ADDR_PRIMARY,isisants_sideA,isisants_arm);
+			error = IsisAntS_setArmStatus(ANTENNA_INDEX, isisants_sideA, isisants_arm);
 
 			// Error check for Arming A Side
 			if(error != 0) {
```

Comment:
All across the file (and any other files): don't compare against `0` because that's a "magic number". Instead use the macro `SUCCESS` (defined as `0` in `RCommon.h`) whenever checking for or returning success